tomviz/MainWindow.cxx

```diff
@@ -690,6 +690,13 @@ void MainWindow::closeEvent(QCloseEvent* e)
       e->ignore();
       return;
     }
+  } else if (ModuleManager::instance().hasDataSources()) {
+    QMessageBox::StandardButton response = QMessageBox::question(
+      this, "Close tomviz?", "Are you sure you want to exit?");
```

Comment:
The title can be "Close?" Repeating the application name isn't needed in my opinion.